Casks/r/radiola.rb

```diff
@@ -0,0 +1,31 @@
+cask "radiola" do
+  version "8.1.0"
+  sha256 "96ecfb960e4cc689a7b5045e6168f91d31906cb27f11d13b28a4a6313124f9b3"
+
+  url "https://github.com/SokoloffA/radiola/releases/download/v#{version}/Radiola-#{version}.dmg"
+  name "Radiola"
+  desc "Internet radio player for the menu bar"
+  homepage "https://github.com/SokoloffA/radiola"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```

Comment:
We should be able to use the built-in update feed for this

```suggestion
  livecheck do
    url "https://sokoloffa.github.io/radiola/feed.xml"
    strategy :sparkle
  end
```

Reply:
Done!